<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>RecorderDocker</class>
 <widget class="QWidget" name="RecorderDocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>369</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>324</width>
        <height>369</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>14</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetSettings" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="layoutSnapshots" stretch="0,0">
            <item>
             <widget class="QLabel" name="labelDirectory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string comment="A folder where recordings are stored">Recordings directory:</string>
              </property>
              <property name="buddy">
               <cstring>buttonBrowse</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonManageRecordings">
              <property name="toolTip">
               <string>Manage recordings...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="folder">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutDir">
            <item>
             <widget class="QLineEdit" name="editDirectory">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonBrowse">
              <property name="toolTip">
               <string>Select the recordings directory...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="folder">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="labelRate">
              <property name="text">
               <string extracomment="This value changes for RealtTimeCapture mode switched to on. It can alter the column width of the labels, if this value is the longest of all labels. Please ensure, that at least one other label has more characters than this one (by stuffing space characters after the colon of one label).">Capture interval:</string>
              </property>
              <property name="buddy">
               <cstring>spinRate</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinRate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="correctionMode">
               <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string comment="Contraction of &quot;second&quot; for interval"> sec.</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelQuality">
              <property name="text">
               <string comment="Compression quality">Quality:</string>
              </property>
              <property name="buddy">
               <cstring>spinQuality</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="KisSliderSpinBox" name="spinQuality" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelResolution">
              <property name="text">
               <string comment="Resolution for recorded frames" extracomment="Please do not remove the spaces after the label text. It ensures to fix the column width if the label text of `labelRate` is set to an other value for RealTimeCaptureMode switched on.">Resolution:         </string>
              </property>
              <property name="buddy">
               <cstring>comboResolution</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="comboResolution">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Original</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Half</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quarter</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelFormat">
              <property name="text">
               <string comment="File format for recorded frames (PNG/JPG)">Format:</string>
              </property>
              <property name="buddy">
               <cstring>comboResolution</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboFormat">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string notr="true">JPEG</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">PNG</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="1">
             <layout class="QHBoxLayout" name="horizontalLayoutThreads">
              <item>
               <widget class="QSlider" name="sliderThreads">
                <property name="toolTip">
                 <string>Number of threads, which should be used for recording</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinThreads">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Number of threads, which should be used for recording</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelThreads">
              <property name="text">
               <string>Threads:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxRealTimeCaptureMode">
            <property name="toolTip">
             <string extracomment="Tooltip for check box">Calculate the Capture Interval from the Video FPS export settings.
Please handle with care, may cause performance issues.</string>
            </property>
            <property name="text">
             <string>Real Time Capture mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxRecordIsolateMode">
            <property name="toolTip">
             <string>Record frames when Isolate Layer or Isolate Group mode is active.
Depending on timelapse speed, this may cause flickering in the resulting video.</string>
            </property>
            <property name="text">
             <string comment="Record when Isolate Layer mode is active">Record in isolate mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAutoRecord">
            <property name="toolTip">
             <string extracomment="Tooltip for check box">Start recording after creating a new image or opening an existing one.</string>
            </property>
            <property name="text">
             <string>Record automatically</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutButtons">
         <item>
          <widget class="QPushButton" name="buttonRecordToggle">
           <property name="text">
            <string>Record</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExport">
           <property name="text">
            <string>Export...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>buttonManageRecordings</tabstop>
  <tabstop>editDirectory</tabstop>
  <tabstop>buttonBrowse</tabstop>
  <tabstop>spinRate</tabstop>
  <tabstop>comboFormat</tabstop>
  <tabstop>spinQuality</tabstop>
  <tabstop>comboResolution</tabstop>
  <tabstop>sliderThreads</tabstop>
  <tabstop>spinThreads</tabstop>
  <tabstop>checkBoxRealTimeCaptureMode</tabstop>
  <tabstop>checkBoxRecordIsolateMode</tabstop>
  <tabstop>checkBoxAutoRecord</tabstop>
  <tabstop>buttonRecordToggle</tabstop>
  <tabstop>buttonExport</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sliderThreads</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinThreads</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinThreads</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderThreads</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
